libraries/postgres/src/Database.js

```diff
@@ -120,6 +120,50 @@ class Database {
     }
   }
 
+  /**
+   * Downgrade this database to the given version and define functions for all
+   * of the methods in that version.  Note that this does not remove unknown
+   * functions.
```

Comment:
What unknown functions is this referring to?

Reply:
For example if a new stored function is defined in version 7, then downgrading to version 6 won't delete it.